ndr_fasttransfer.c

```diff
@@ -572,15 +571,14 @@ static int ndr_parse_FastTransferStream(struct ndr_print *ndr, DATA_BLOB buf)
 	ndr_set_flags(&ndr_pull->flags, LIBNDR_FLAG_NOALIGN);
 	ndr_set_flags(&ndr->flags, LIBNDR_FLAG_NOALIGN);
 
-	while (ndr_pull->offset < ndr_pull->data_size -1) {
-		offset = ndr_parse_FastTransferElement(mem_ctx, ndr, ndr_pull);
-		if (offset < 0) {
+	while (ndr_pull->offset < ndr_pull->data_size - 1) {
+		ret = ndr_parse_FastTransferElement(mem_ctx, ndr, ndr_pull);
+		if (ret != 0) {
```

Comment:
The documentation on `ndr_parse_FastTransferElement` says that `ret` > 0 is ok, not an error. If the documentation is wrong, I'd suggest to modify it. If it is right, please change the check :).